recipes/netcdf-scm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "netcdf-scm" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a9e92982ea95ebbf12fc03abcc1659127e9bd7c8446841fd9dc39aa878f7ab4
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - iris
+    - netcdftime
+    - nc-time-axis
+
+test:
+  imports:
+    - netcdf_scm
+  source_files:
+    - tests/*
+  commands:
+    - conda install -y pip; pip install git+https://github.com/openclimatedata/pymagicc.git@master pandas
```

Comment:
We don't expect `pip install` to succeed there :-/
Also, a `conda install` during a package build is not OK and should not work.

Reply:
ok I'll put making my dependencies conda forge installable on the to do list then